component-samples/device/src/main/java/org/fidoalliance/fdo/sample/ConformanceDeviceModule.java

```diff
@@ -1,50 +1,120 @@
 import org.fidoalliance.fdo.protocol.Mapper;
 import org.fidoalliance.fdo.protocol.dispatch.ServiceInfoModule;
 import org.fidoalliance.fdo.protocol.dispatch.ServiceInfoSendFunction;
+import org.fidoalliance.fdo.protocol.entity.ConformanceData;
+import org.fidoalliance.fdo.protocol.entity.OnboardingConfig;
+import org.fidoalliance.fdo.protocol.message.AnyType;
+import org.fidoalliance.fdo.protocol.message.DevModList;
+import org.fidoalliance.fdo.protocol.message.Guid;
+import org.fidoalliance.fdo.protocol.message.ServiceInfo;
 import org.fidoalliance.fdo.protocol.message.ServiceInfoKeyValuePair;
 import org.fidoalliance.fdo.protocol.message.ServiceInfoModuleState;
+import org.fidoalliance.fdo.protocol.message.ServiceInfoQueue;
+import org.fidoalliance.fdo.protocol.message.To2AddressEntries;
+import org.fidoalliance.fdo.protocol.serviceinfo.DevMod;
 import org.fidoalliance.fdo.protocol.serviceinfo.FidoAlliance;
+import org.hibernate.Session;
+import org.hibernate.Transaction;
 
-public class ConformanceDeviceModule implements ServiceInfoModule {
+public class ConformanceOwnerModule implements ServiceInfoModule {
 
+<<<<<<< HEAD
+=======
   private final LoggerService logger = new LoggerService(DeviceApp.class);;
+
+>>>>>>> added fdo_sys modlue
   @Override
   public String getName() {
     return FidoAlliance.NAME;
   }
 
   @Override
   public void prepare(ServiceInfoModuleState state) throws IOException {
+    state.setExtra(AnyType.fromObject(new ServiceInfoQueue()));
 
   }
 
   @Override
   public void receive(ServiceInfoModuleState state, ServiceInfoKeyValuePair kvPair)
       throws IOException {
-
     switch (kvPair.getKey()) {
-      case FidoAlliance.ACTIVE:
-        logger.info(FidoAlliance.ACTIVE+ " = "
-            + Mapper.INSTANCE.readValue(kvPair.getValue(),Boolean.class));
-        state.setActive(true);
-        break;
-      case FidoAlliance.DEV_CONFORMANCE:
-        if (state.isActive()) {
-          logger.info(FidoAlliance.DEV_CONFORMANCE + " = "
-              + Mapper.INSTANCE.readValue(kvPair.getValue(),String.class));
-          break;
+      case DevMod.KEY_MODULES: {
+        DevModList list =
+            Mapper.INSTANCE.readValue(kvPair.getValue(), DevModList.class);
+        for (String name : list.getModulesNames()) {
+          if (name.equals(FidoAlliance.NAME)) {
+            state.setActive(true);
+            ServiceInfoQueue queue = state.getExtra().covertValue(ServiceInfoQueue.class);
+            ServiceInfoKeyValuePair activePair = new ServiceInfoKeyValuePair();
+            activePair.setKeyName(FidoAlliance.ACTIVE);
+            activePair.setValue(Mapper.INSTANCE.writeValue(true));
+            queue.add(activePair);
+            getConformance(state.getGuid(), queue);
+            state.setExtra(AnyType.fromObject(queue));
+   
+          }
         }
+<<<<<<< HEAD
+      }
+      break;
+=======
+>>>>>>> added fdo_sys modlue
+      default:
+        break;
```

Comment:
another merge conflict

Reply:
this was fix in commit a0cbcb9f3a1801f6e4631c23d5803face1f4e9ce